src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/useOtherRelatedInvoiceLines.js

```diff
@@ -14,17 +14,41 @@ import {
 
 import { INVOICE_LINE_API } from '../../../common/constants';
 
-export const useOtherRelatedInvoiceLines = (invoiceLineId, poLineId) => {
+function getSortQuery(sorting) {
```

Comment:
probably you can reuse `buildSortingQuery` from `stripes-acq-components`

Reply:
introducing sorting on BE side you introduce issues, not everything can be sorted there, as I pointed earlier, you need client side pagination

Reply:
@NikitaSedyx after discussing this with Yury, we decided to use BE pagination and sorting, since sorting will not work probably.
Or did you mean, We need to fetch all invoices and use pagination in FE so that sorting and pagination works correctly ?

Reply:
Yeah, really, it seems we can't sort by foreign table fields...

Reply:
client side pagination =  to fetch all invoices and use FE pagination so that sorting and pagination work correctly

Reply:
you don't need to change fetching mechanism, all you need is Pagination component and hook to set up pages + saving visible portion of invoices